rpc/src/helpers.rs

```diff
@@ -569,11 +569,35 @@ pub(crate) fn current_time_timestamp() -> TimeStamp {
 
 pub(crate) async fn create_ffi_json_request(req: Request<Body>) -> Result<JsonRpcRequest, failure::Error> {
     let context_path = req.uri().path_and_query().unwrap().as_str().to_string();
+    let meth = RpcMethod::try_from(req.method().to_string().as_str()).unwrap(); // TODO: handle correctly
+    let content_type = None; // TODO: get from request
```

Comment:
@tizoc what about this content-type?

Reply:
@bkontur right now it is not used, and on the OCaml side it defaults to JSON encoding when the value is `None`. There is something else missing I just noticed, the "Accept" header (so, content-type => what to parse the input as, "accept" => what output encoding to use).

Reply:
@bkontur rechecking the code, I now remember why that part was a `TODO`. Currently every function in there has a `serde_json::value::Value` return type, so to be able to make use of the content type specification headers, this would have to change too. I will update the code to pass the value from the headers, but for now that will be ignored.

Reply:
ok, ok

Reply:
https://github.com/simplestaking/tezedge/pull/318/files#diff-b4be62d44950d767fd17e69305f99558119c01ab676c79b557041ee102faa398R570-R589 <- these values are ignored on the OCaml side for now, but the information is passed on now.

Reply:
ok, cool, I see you also renamed: JsonRpcRequest -> RpcRequest, nice :)